integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/management/FindEvents.feature

```diff
@@ -48,3 +49,11 @@ Feature: SmartMetering Management - Find Events
       | EndDate              | 2015-09-05T00:00:00.000Z |
     Then 29 mbus events should be returned
       | DeviceIdentification | TEST1024000000001 |
+
+  Scenario: find power failure events from a device
+    When receiving a find power failure events request
+      | DeviceIdentification | TEST1024000000001        |
+      | BeginDate            | 1970-01-01T00:00:00.000Z |
```

Comment:
This is some date range. Can it be more specific?

Reply:
The date range is always from the beginning of (java) time to the current date, today.